synapse/tests/test_lib_storm.py

```diff
@@ -649,6 +649,9 @@ def test_storm_guid_stablegen(self):
             self.notin('.new', node1[1])
             self.eq(node0[0], node1[0])
 
+            node2 = core.eval('addnode(ps:hasnetuser, ((guidname="bob gray"),vertex.link/pennywise))')[0]
```

Comment:
I would add another test with `ou:hasnetuser` which is a comptype. I get the same error for each locally.

Reply:
I just tried it out on my local, it works for `sepr` but does not work for `comp`

Reply:
added

Reply:
thanks!